### PR TITLE
chore: Update read environment variables guide update with new .env guidance

### LINKED ISSUE #6140: Update how-to-read-environment-variables-from-nodejs.md  with new `.env` guidance
Update pages/en/learn/command-line/how-to-read-environment-variables-from-nodejs.md

This `.env` file addition should be a separate issue - otherwise this PR will be unresolved for longer.

_Originally posted by @bmuenzenmeyer in https://github.com/nodejs/nodejs.org/pull/6132#discussion_r1403442791_

### PR DESCRIPTION
**Pull request by nikhilbhatt**
## Description
Fixes #6140 

### Check List

- [x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [ ] I have run `npx turbo lint` to ensure the code follows the style guide. And run `npx turbo lint:fix` to fix the style errors if necessary.
- [ ] I have run `npx turbo format` to ensure the code follows the style guide.
- [ ] I have run `npx turbo test` to check if all tests are passing.
- **NA** I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

**Comment by AugustinMauroy** [comment]
Personally, I would have removed the part that talks about the `dotenv` package.

**Review by AugustinMauroy — APPROVED** [review]
LGTM !!! 
🎉 thanks for your first contribution !!

**Review by aymen94 — APPROVED** [review]

File: pages/en/learn/command-line/how-to-read-environment-variables-from-nodejs.md [hunk 1]
```diff
@@ -27,25 +27,33 @@ process.env.USER_KEY; // "foobar"
...
-process.env.USER_KEY; // "foobar"
-process.env.NODE_ENV; // "development"
+Run `app.js` file with environment variables set in `.env` file.
+
+```bash
+node --env-file=.env app.js
 ```
 
-> You can also run your js file with `node -r dotenv/config index.js` command if you don't want to import the package in your code.
+This command loads all the environment variables from `.env` file, making them available to application on `process.env`
```
Comment (bmuenzenmeyer) [inline pages/en/learn/command-line/how-to-read-environment-variables-from-nodejs.md]: grammar

```suggestion
This command loads all the environment variables from the `.env` file, making them available to the application on `process.env`
```

File: pages/en/learn/command-line/how-to-read-environment-variables-from-nodejs.md [hunk 2]
```diff
@@ -27,25 +27,33 @@ process.env.USER_KEY; // "foobar"
...
+
+Also, You can pass multiple `--env-file` arguments. Subsequent files override pre-existing variables defined in previous files.
```
Comment (bmuenzenmeyer) [inline pages/en/learn/command-line/how-to-read-environment-variables-from-nodejs.md]: capitlization

```suggestion
Also, you can pass multiple `--env-file` arguments. Subsequent files override pre-existing variables defined in previous files.
```

**Review by bmuenzenmeyer — APPROVED** [review]

**Pull request merged**